runner/runs/quad_multi_benchmark.py

```diff
@@ -0,0 +1,34 @@
+from runner.run_description import RunDescription, Experiment, ParamGrid
+
+_params = ParamGrid([
+    ('seed', [0000, 1111, 2222, 3333, 4444, 5555, 6666, 7777]),
+    ('quads_num_agents', [4]),
```

Comment:
What's the point of specifying these parameters here in a gridsearch?
They could be just regular command line parameters.
If you want this for readability, consider just splitting the CMD string parts and += some special parameters in the end.

Because currently these parameters are here, they will be made a part of the filenames in tensorboard etc. which will result in very long and nonsensical names.

Reply:
Sure, I will modify it.